lib/ansible/modules/packaging/os/dnf.py

```diff
@@ -616,6 +617,26 @@ def _base(self, conf_file, disable_gpg_check, disablerepo, enablerepo, installro
                 results=[],
                 rc=1
             )
+        except AttributeError as e:
+            if 'demands' in to_text(e):
```

Comment:
I wonder if including a comment that includes a sample of the line we expect to be matched maybe helpful.